ci/test/pipeline.template.yml

```diff
@@ -387,7 +387,7 @@ steps:
   - id: pg-cdc
     label: Postgres CDC tests
     depends_on: build-x86_64
-    timeout_in_minutes: 30
+    timeout_in_minutes: 45
```

Comment:
The test runs in 8 min without Coverage, so this value was more than sufficient.